src/api/spotify/player.ts

```diff
@@ -2,15 +2,28 @@ import { useSpotifyApi } from 'components/providers/FetchProvider'
 import { useQuery, UseQueryResult } from 'react-query'
 import SpotifyWebApi from 'spotify-web-api-node'
 
-const getRecentlyPlayed = async (api: SpotifyWebApi): Promise<unknown> => {
+const getTrack = async (api: SpotifyWebApi, trackId: string)
+: Promise<SpotifyApi.SingleTrackResponse> => {
+  const { body } = await api.getTrack(trackId)
+  return body
+}
+
+const useTrack = (trackId: string)
+: UseQueryResult<SpotifyApi.SingleTrackResponse, unknown> => {
+  const spotifyApi = useSpotifyApi()
+  return useQuery(['track', trackId], () => getTrack(spotifyApi, trackId))
+}
```

Comment:
fajnie by było jakbyśmy się trzymali podziału w plikach taki jaki mają w api tzn. wydzieliłbym to do pliku tracks.ts (bo wg dokumentacji te endpointy znajdują się w Tracks API)